### ISSUE TITLE
Stopped working on Firefox 42 in Android 5.x and 6.x.

### ISSUE THREAD

**Issue by BastianBalthasarBux** [ComputerWolf/SlickNav#115]
The Menu-Button is not touch-sensitive anymore. Worked before.
Also on slicknav.com not working, therefore most likely not an integration error.

**Comment by BastianBalthasarBux** [comment]
Ok, I investigated a bit more.
It is really funny.
The click on the burger is not registered anymore. Although the correct link is in place. If I tap very long on the burger, it opens up the window 'open link in new tab' etc. the correct anchor is displayed. I crosstested by changing the href="#" to something else.
so it is not a positioning problem or similar.
As soon as I put in some text for the label, it works. But just on the label, not on the burger-icon.
Astonishing, or better: weird ...

All these findings just concern the Firefox 42 on Android. On chrome and opera on the same devices it works as expected!
